tests/test_urls.py

```diff
@@ -52,7 +52,7 @@ def test_404_page_not_created(app, status, warning):
     path = app.outdir / '404.html'
     assert not path.exists()
 
-@pytest.mark.sphinx(srcdir=srcdir)
+@pytest.mark.sphinx(srcdir=srcdir, confoverrides={'html_theme': 'alabaster'})
```

Comment:
Can you put a comment here explaining why we are defining the theme explicitly here and link to the Sphinx PR that performs the change?